pkg/cmd/init.go

```diff
@@ -27,7 +27,10 @@ import (
 	"github.com/openshift/microshift/pkg/util/cryptomaterial"
 )
 
-var microshiftDataDir = config.GetDataDir()
+var (
+	microshiftDataDir = config.GetDataDir()
+	nodeIP            = config.GetNodeIP()
```

Comment:
instead of defining that globally every time, could we perhaps create a structure that could be passed on to the various init functions alongside the configuration?